cmd/controller-manager/app/controller-manager.go

```diff
@@ -58,7 +59,6 @@ member clusters and does the necessary reconciliation`,
 			if verFlag {
 				os.Exit(0)
 			}
-			PrintFlags(cmd.Flags())
```

Comment:
Why does this need to be removed?

Reply:
this has already been print in the "k8s.io/apiserver/pkg/util/flag", see [this](https://github.com/kubernetes-sigs/federation-v2/blob/master/vendor/k8s.io/apiserver/pkg/util/flag/flags.go#L51)